Tests/Fuzzers/CMakeLists.txt

```diff
@@ -1,3 +1,5 @@
-add_executable(FuzzTarget FuzzTarget.cpp)
+add_executable(FuzzPacketParse FuzzPacketParse.cpp)
```

Comment:
I think the fact we're changing the executable name makes the regression tests fail:
https://cirrus-ci.com/task/5398413525647360